.travis.yml

```diff
@@ -13,7 +13,9 @@ perl:
     - "5.16"
 
 before_install:
-    - eval $(curl https://travis-perl.github.io/init)
+      # quoting preserves newlines in the script and then avoid error if the
+      # script contains comments
+    - eval "$(curl https://travis-perl.github.io/init)"
```

Comment:
btw if anyone knows why we use the `--auto` parameter in the other repos, I'm curious to know about it.

Reply:
No, but it seems to work without `--auto`. The `init` script must be investigated in more details to understand what `--auto` adds.